verifiers/internal/gha/verifier.go

```diff
@@ -34,23 +38,11 @@ func (v *GHAVerifier) IsAuthoritativeFor(builderID string) bool {
 	return strings.HasPrefix(builderID, "https://github.com/")
 }
 
-// VerifyArtifact verifies provenance for an artifact.
-func (v *GHAVerifier) VerifyArtifact(ctx context.Context,
-	provenance []byte, artifactHash string,
+func verifyEnvAndCert(env *dsse.Envelope,
```

Comment:
remind me: do we verify that the leaf cert is signed by Fulcio?

Reply:
of course we do https://github.com/slsa-framework/slsa-verifier/blob/main/verifiers/internal/gha/rekor.go#L375
Closing.